common/src/limits.rs

```diff
@@ -17,7 +17,7 @@ pub struct Limits {
     #[serde(skip_deserializing)]
     rds_access: bool,
     /// The quantity of RDS instances this user can provision.
-    pub(crate) rds_quota: u32,
+    pub rds_quota: u32,
```

Comment:
BTW I created a method on Limits to set the quota, if we're setting the field directly now we could remove it, I don't think it's used anywhere.

Reply:
Ah, thanks!! I missed that. Will update the code to use the method instead

Reply:
I see from the failing tests that we'd need a getter as well now, since we need to read the quota. So we either make it pub, or add a getter method, and the setter should actually be renamed `set_rds_quota` in accordance with Rust naming convention.  :sweat_smile:

Reply:
Haha, I just saw the same. Will have a proper setter in a bit...